pkg/rancher-desktop/main/commandServer/settingsValidator.ts

```diff
@@ -492,6 +492,83 @@ export default class SettingsValidator {
     return false;
   }
 
+  protected checkExtensions(
+    mergedSettings: Settings,
+    currentValue: Record<string, string>,
+    desiredValue: any,
+    errors: string[],
+    fqname: string,
+  ): boolean {
+    if (_.isEqual(desiredValue, currentValue)) {
+      // Accept no-op changes
+      return false;
+    }
+
+    if (typeof desiredValue !== 'object' || !desiredValue) {
+      errors.push(`${ fqname }: "${ desiredValue }" is not a valid mapping`);
+
+      return false;
+    }
+
+    /**
+     * makeRE is a tagged template for making regular expressions with /x (i.e.
+     * ignoring any whitespace within the regular expression itself).
+     * @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#tagged_templates
+     */
+    function makeRE(strings: TemplateStringsArray, ...substitutions: any[]) {
```

Comment:
The `makeRE` function and all uses of it are constants and can be hoisted outside this instance method

Reply:
Moved to global scope in 0d30157beb3c1da83b549608177f080269872f57.